### PR TITLE
fix runtime error

### PR DESCRIPTION
**Pull request by wangsai-silence**
there no filed named transactionLogIndex in log, use logIndex.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -37,7 +37,7 @@ function parseReceiptEvents(abi, address, receipt) {
       delete log.data
       delete log.topics
 
-      const eventNumber = Number.parseInt(log.transactionLogIndex.substr(2), 16)
+      const eventNumber = log.logIndex
```
Comment (gabmontes) [inline src/index.js:40]: The property `transactionLogIndex` is not standard. It is not described in the official Ethereum JSON RPC API, Web3 docs or returned by Geth at all. Seems to be returned only by OpenEthereum nodes. @wangsai-silence what is your setup?

So `logIndex` is the correct property but if for some reason other Ethereum node implementations don't return it and return `transactionLogIndex` instead, I'm ok in trying to parse `transactionLogIndex` if `logIndex` is not present.

**Review by gabmontes — CHANGES_REQUESTED** [review]
@wangsai-silence thanks for submitting this PR. See my comments on the proposed change.

File: src/index.js (hunk 1 above)
Comment (wangsai-silence) [reply src/index.js:40]: I use go-ethereum. And it defines here [go-ethereum/log.go](https://github.com/ethereum/go-ethereum/blob/7b748e550a3234c5240427d0164a058829dc83dc/core/types/log.go#L51).  Maybe we can check both logIndex and transactionLogIndex for compatibility.

Comment (wangsai-silence) [reply src/index.js:40]: Should I just close this PR and commit a new one?

Comment (gabmontes) [reply src/index.js:40]: @wangsai-silence I totally misread your change. It is correct. Original code is actually using a non-standard property. Thanks for pointing this out!

**Review by gabmontes — APPROVED** [review]

**Pull request merged**